e2e/wdio/core/tests/avatar-group.e2e-spec.ts

```diff
@@ -0,0 +1,34 @@
+import { AvatarGroupPo } from '../pages/avatar-group.po';
+import { click, doesItExist, getElementSize, scrollIntoView, waitForPresent } from '../../driver/wdio';
+
+describe('Avatar test suite', function() {
+    const avatarGroupPage = new AvatarGroupPo();
+
+    beforeAll(() => {
+        avatarGroupPage.open();
+    }, 1);
+
+    it('should have details popup on click', () => {
+        click(avatarGroupPage.firstExampleAvatar);
+        waitForPresent(avatarGroupPage.usedDetailsPopup);
+        expect(doesItExist(avatarGroupPage.usedDetailsPopup)).toBe(true);
+
+        scrollIntoView(avatarGroupPage.secondExampleAvatar);
+        click(avatarGroupPage.secondExampleAvatar);
+        waitForPresent(avatarGroupPage.usedGroupDetailsPopup);
+        expect(doesItExist(avatarGroupPage.usedGroupDetailsPopup)).toBe(true);
+    });
+
```

Comment:
I think 1-2 more cases are needed
when you click on group type it opens a popup with individual users (you check this now), each user is clickable and opens an individual user card. Should check this. also back btn functionality on this user card